deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -1,184 +1,187 @@
+```
+
+## Testing the deployed solution
+
+Once the solution is deployed, the easiest way to test the solution is to access the `migration-console` service container and run an opensearch-benchmark workload through to simulate incoming traffic, as the following steps illustrate
+
+```shell
+# Exec into container
+./accessContainer.sh migration-console dev us-east-1
+
+# Run opensearch-benchmark workload (i.e. geonames, nyc_taxis, http_logs)
+./runTestBenchmarks.sh
+```
+
+After the benchmark has been run, the indices and documents of the source and target clusters can be checked from the same migration-console container to confirm
+```shell
+# Check doc counts and indices for both source and target cluster
+./catIndices.sh
+```
+
+## Kicking off Fetch Migration
+
+* First, access the Migration Console container
+
+```shell
+# ./accessContainer.sh migration-console STAGE REGION
+./accessContainer.sh migration-console dev us-east-1
+```
+
+* Execute the ECS run task command stored in the container's environment
+    * The status of the ECS Task can be monitored from the AWS Console. Once the task is in the `Running` state, logs and progress can be viewed via CloudWatch.
+```shell
+# This will print the needed ECS command
+echo $FETCH_MIGRATION_COMMAND
+
+# Paste the above printed command into the terminal to kick off
+```
+
+The pipeline configuration file can be viewed (and updated) via AWS Secrets Manager.
+
+
+## Tearing down CDK
+To remove all the CDK stack(s) which get created during a deployment we can execute a command similar to below
+```shell
+# For demo context
+cdk destroy "*" --c contextId=demo-deploy
+```
+Or to remove an individual stack from the deployment we can execute
+```shell
+# For demo context
+cdk destroy migration-console --c contextId=demo-deploy
 ```
 Note that the default retention policy for the OpenSearch Domain is to RETAIN this resource when the stack is deleted, and in order to delete the Domain on stack deletion the `domainRemovalPolicy` would need to be set to `DESTROY`. Otherwise, the Domain can be manually deleted through the AWS console or through other means such as the AWS CLI.
 
+## How to run multiple Traffic Replayer scenarios
+The project supports running distinct Replayers in parallel, with each Replayer sending traffic to a different target cluster. This functionality allows users to test replaying captured traffic to multiple different target clusters in parallel. Users are able to provide the desired configuration options to spin up a new OpenSearch Domain and Traffic Replayer while using the existing Migration infrastructure that has already been deployed.
+
+To give an example of this process, a user could decide to configure an additional Replayer and Domain for the demo setup in the `cdk.context.json` by configuring a new context block like below. **Note**: `addOnMigrationDeployId` is a required field to allow proper naming of these additional resources.
```

Comment:
Not yet but we have a task that will allow using an existing target endpoint instead of requiring one to be created: https://opensearch.atlassian.net/browse/MIGRATIONS-1361. Right now we don't have any section which explains the stacks that get created and I'm still looking for a valuable way to tell that story